src/internal/components/LayoutEditor.tsx

```diff
@@ -64,7 +64,7 @@ export const LayoutEditor = (props: LayoutEditorProps) => {
    * Apply the themes from Content
    */
   useEffect(() => {
-    if (!themeConfig) {
+    if (!themeConfig || !templateMetadata?.themeEntityId) {
```

Comment:
I don't think this is the appropriate spot for this check. If anything the check should happen in useThemeLocalStorage which would return "" and is already handled by line 74.